### PR TITLE
Various material profiles with all available colors added

### PR DESCRIPTION
**Pull request by x40-Community**
We have a large material and print profile database from the X40 Community project that we would like to make available to everyone.

### CODE REVIEW

File: Verbatim_PETG_Black_175.xml.fdm_material [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura" version="1.3">
+  <metadata>
+    <name>
+      <brand>Verbatim</brand>
+      <material>PET-G</material>
```
Comment (nallath) [inline Verbatim_PETG_Black_175.xml.fdm_material:6]:
```suggestion
      <material>PETG</material>
```

PETG is the material type that is used everywhere else.

File: Verbatim_PETG_Blue_175.xml.fdm_material [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura" version="1.3">
+  <metadata>
+    <name>
+      <brand>Verbatim</brand>
+      <material>PET-G</material>
```
Comment (nallath) [inline Verbatim_PETG_Blue_175.xml.fdm_material:6]:
```suggestion
      <material>PETG</material>
```

File: Verbatim_PETG_Bluetransparent_175.xml.fdm_material [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura" version="1.3">
+  <metadata>
+    <name>
+      <brand>Verbatim</brand>
+      <material>PET-G</material>
```
Comment (nallath) [inline Verbatim_PETG_Bluetransparent_175.xml.fdm_material:6]:
```suggestion
      <material>PETG</material>
```

File: Verbatim_PETG_Clear_175.xml.fdm_material [hunk 4]
```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura" version="1.3">
+  <metadata>
+    <name>
+      <brand>Verbatim</brand>
+      <material>PET-G</material>
+      <color>Clear</color>
+      <label>Clear PET-G</label>
```
Comment (nallath) [inline Verbatim_PETG_Clear_175.xml.fdm_material:8]:
```suggestion
      <label>Clear PETG</label>
```

File: Verbatim_PETG_Greentransparent_175.xml.fdm_material [hunk 5]
```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura" version="1.3">
+  <metadata>
+    <name>
+      <brand>Verbatim</brand>
+      <material>PET-G</material>
```
Comment (nallath) [inline Verbatim_PETG_Greentransparent_175.xml.fdm_material:6]:
```suggestion
      <material>PETG</material>
```

File: Verbatim_PETG_Red_175.xml.fdm_material [hunk 6]
```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura" version="1.3">
+  <metadata>
+    <name>
+      <brand>Verbatim</brand>
+      <material>PET-G</material>
```
Comment (nallath) [inline Verbatim_PETG_Red_175.xml.fdm_material:6]:
```suggestion
      <material>PETG</material>
```

File: Verbatim_PETG_Redtransparent_175.xml.fdm_material [hunk 7]
```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura" version="1.3">
+  <metadata>
+    <name>
+      <brand>Verbatim</brand>
+      <material>PET-G</material>
```
Comment (nallath) [inline Verbatim_PETG_Redtransparent_175.xml.fdm_material:6]:
```suggestion
      <material>PETG</material>
```

File: Verbatim_PETG_White_175.xml.fdm_material [hunk 8]
```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura" version="1.3">
+  <metadata>
+    <name>
+      <brand>Verbatim</brand>
+      <material>PET-G</material>
+      <color>White RAL 9003</color>
+      <label>White PET-G</label>
```
Comment (nallath) [inline Verbatim_PETG_White_175.xml.fdm_material:8]:
```suggestion
      <label>White PETG</label>
```

**Review by nallath — CHANGES_REQUESTED** [review]

**Review by nallath — DISMISSED** [review]

**Comment by Ghostkeeper** [comment]
There is a note here about material types that may require changes to this PR: https://github.com/Ultimaker/Cura/pull/10345#issuecomment-972968258

**Pull request closed without merging**